docs/operations/storage/README.md

```diff
@@ -0,0 +1,77 @@
+# Loki Storage
```

Comment:
I think somewhere in this file provide a list of supported stores for the index and chunks. Currently, that information is only available in the architecture doc.